index.bs

```diff
@@ -53,6 +61,73 @@ dictionary StorageBucketOptions {
 
 <h3 id="storage-bucket-open">Creating a bucket</h3>
 
+<div algorithm>
+
+  The <dfn method for="StorageBucketManager">open(|name|, |options|)</dfn> method steps are:
+
+  1. Let |environment| be [=/this=]'s [=/relevant settings object=].
+
+  1. Let |storageKey| be the result of running [=obtain a storage key=] given |environment|.
+
+      If failure is returned, then [=exception/throw=] a "{{SecurityError}}" {{DOMException}} and abort these steps.
```

Comment:
I think this line should have a `1.` in front, also maybe phrase it as "If |storageKey| is failure" (?)